packages/dev/docs/pages/blog/building-a-combobox.mdx

```diff
@@ -0,0 +1,94 @@
+
+### Differences in screen readers
+
+Just like browsers, screen readers can be quite varied in behavior and may require deliberate tooling to ensure a consistent degree of functionality.
+
+#### NVDA
+
+Initially, our ComboBox would automatically focus the first item in the menu whenever opened so the user wouldn't have to issue another keypress to begin menu navigation. However, when we tested the ComboBox using NVDA we discovered that
+character deletions and text cursor movement in the ComboBox input weren't being announced at all. NVDA would only resume announcing if no menu items were being focused, so we ended up clearing menu item focus on any changes to the input text or
+left/right arrow key presses.
+
+#### VoiceOver
+
+VoiceOver as a whole doesn't seem to work very well with `aria-activedescendant`, failing to announce when the focused ComboBox item changed in a variety of situations. In Safari, changes to the `aria-activedescendant` aren't announced
+as long as the input is empty. In Chrome, VoiceOver only announces changes to `aria-activedescendant` if `aria-selected="true"` is applied to the focused item. In Firefox, VoiceOver only announces the first item you focused. If the list of options is organized into separate sections,
+VoiceOver doesn't announce changes to the focused item for all browsers.
+
+Announcing item focus wasn't the only issue encountered. VoiceOver doesn't announce the number of options currently available in the menu or the current section title, information that an end user would find helpful when navigating the ComboBox menu.
+When selecting an option from the menu, an announcement of the item's name is made but the fact that it is now "selected" is omitted.
+
+As a workaround, we made use of [ARIA live-regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions). When the ComboBox's menu is opened for the first time, we create two visually hidden `aria-live` regions in the DOM with two `div`s, one with `aria-live="polite"` and the other `aria-live="assertive"`
+so that we have control over how the message should be announced to the user. Then whenever updates occur in the ComboBox, we can have the screen reader announce custom messages by updating the contents of said `div`s. This allows us to fill in the announcement gaps mentioned previously as well as standardize the
+announcement experience over each of the browsers. If you are interested in using this LiveAnnouncer yourself, check out `LiveAnnouncer` in `@react-aria/live`.
+
+## Conclusion
+
+If there is one takeaway, it is that cross browser and device testing is extremely important when building a component. A solution that may work in one browser and device combo may result in a broken experience in another. Browsers, devices, and assistive technologies each have their own behavioral quirks that often are only made apparent
+after thorough testing.
```

Comment:
My own personal takeaway (albeit feels a bit obvious). Kinda wanted to say something along the line of 'combobox is hard' but ehh

Reply:
Yeah I think saying combobox is harder than it looks is reasonable as well. 😉 

You should also add something like "The useComboBox hook handles all of this complexity and has been tested across a wide variety of devices, browsers, and assistive technology combinations. If you're building your own combo box component for your design system, check it out!". Feel free to re-word.